src/bin/server.ts

```diff
@@ -10,10 +10,21 @@ interface Args {
   _: Array<string>;
   config?: string;
   "mock-driver": boolean;
+  "websocket-port": number;
 }
 
 (async () => {
-  const args = parseArgs<Args>(["_", "config", "mock-driver"]);
+  const args = parseArgs<Args>([
+    "_",
+    "config",
+    "mock-driver",
+    "websocket-port",
+  ]);
+
+  let wsPort = 3000;
+  if (args["websocket-port"]) {
+    wsPort = args["websocket-port"];
```

Comment:
`parseArgs` only returns string, no? Shouldn't we convert it to a number first and verify that works?

Reply:
It appears it casts it correctly when the input is a number but accepts a string response (this is me spitting out the args after parsing):
```
$ node_modules/.bin/ts-node src/bin/server.ts --port 3000
{ _: [], port: 3000 }
$ node_modules/.bin/ts-node src/bin/server.ts --port "3000"
{ _: [], port: 3000 }
$ node_modules/.bin/ts-node src/bin/server.ts --port abc
{ _: [], port: 'abc' }
```